tiatoolbox/decorators/multiproc.py

```diff
@@ -1,21 +1,35 @@
 """
-This file defines multiprocessing decorators required by the tiatoolbox.
+Multiprocessing decorators required by the tiatoolbox.
```

Comment:
Inconsistency with spacing at end of docstring- do we want to add blank space at the end of docstring like others?

Reply:
Updated as defined in toolbox notebook.